### PR TITLE
Removes `verify_url` from `ExternalWallet`.

### PR DESCRIPTION
**Pull request by szilardszaloki**
Resolves https://github.com/brave/brave-browser/issues/23682.

## Submitter Checklist:

- [x] I confirm that no security/privacy review [is needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or that I have [requested](https://github.com/brave/security/issues/new/choose) one
- [x] There is a [ticket](https://github.com/brave/brave-browser/issues) for my issue
- [x] Used Github [auto-closing keywords](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) in the PR description above
- [x] Wrote a good [PR/commit description](https://google.github.io/eng-practices/review/developer/cl-descriptions.html)
- [x] Squashed any review feedback or "fixup" commits before merge, so that history is a record of what happened in the repo, not your PR
- [x] Added appropriate labels (`QA/Yes` or `QA/No`; `release-notes/include` or `release-notes/exclude`; `OS/...`) to the associated issue
- [x] Checked the PR locally: `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests`, `npm run lint`, `npm run gn_check`, `npm run tslint`
- [x] Ran `git rebase master` (if needed)

## Reviewer Checklist:

- [ ] A security review [is not needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or a link to one is included in the PR description
- [ ] New files have MPL-2.0 license header
- [ ] Adequate test coverage exists to prevent regressions
- [ ] Major classes, functions and non-trivial code blocks are well-commented
- [ ] Changes in component dependencies are properly reflected in `gn`
- [ ] Code follows the [style guide](https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/c++/c++.md)
- [ ] Test plan is specified in PR before merging

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on
- [ ] All relevant documentation has been updated, for instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-Compatibility-Exceptions-in-Brave
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide
  - [ ] https://github.com/brave/brave-browser/wiki/P3A

### CODE REVIEW

File: components/brave_rewards/resources/shared/lib/external_wallet.ts [hunk 1]
```diff
@@ -99,7 +99,7 @@ export function externalWalletFromExtensionData (
     links: {
       account: String(data.accountUrl || ''),
       addFunds: String(data.addUrl || ''),
-      completeVerification: String(data.verifyUrl || ''),
+      completeVerification: String(data.loginUrl || ''),
```
Comment (zenparsing) [inline components/brave_rewards/resources/shared/lib/external_wallet.ts:102]: I'm thinking that we can probably get rid of this field entirely, and also the `complete-verification` option in `ExternalWalletAction` (from `external_wallet_action.ts`).

Comment (szilardszaloki) [reply components/brave_rewards/resources/shared/lib/external_wallet.ts:102]: Yeah, that's what I was thinking, too. Wanted to reach out to you to discuss. 🙂

Comment (szilardszaloki) [reply components/brave_rewards/resources/shared/lib/external_wallet.ts]: Resolved in 6379b0a4c4ace23910d8dd2694aaf9ae8a335426.

**Review by emerick — APPROVED** [review]
LGTM

**Review by zenparsing — APPROVED** [review]

**Review by deeppandya — APPROVED** [review]
Android changes LGTM

**Pull request merged**